rules/windows/initial_access_dns_server_overflow.toml

```diff
@@ -0,0 +1,51 @@
+[metadata]
+creation_date = "2020/07/16"
+ecs_version = ["1.5.0"]
+maturity = "production"
+updated_date = "2020/07/16"
+
+[rule]
+author = ["Elastic"]
+description = """
+A specially crafted DNS SIG request can manipulate a known vulnerability in some Windows DNS servers which result in
+Remote Code Execution (RCE) or a Denial of Service (DoS) from crashing the service.
+"""
+false_positives = [
+    """
+    Environments that leverage DNS responses over 60k bytes will identify false positives.
+    """,
+]
+index = ["packetbeat-*", "filebeat-*"]
+language = "kuery"
+license = "Elastic License"
+name = "Malicious DNS SIG Request"
```

Comment:
This rule does not identify "Malicious DNS SIG Requests"

I'd prefer something like "Abnormally Large DNS Response" for accuracy, or "Attempted DNS Server Exploitation" if we want to make sure this rule is properly triaged with the urgency I think it needs.

UPDATE:  "DNS Server Overflow Attempt" also sounds good and aligns with the filename.

Reply:
Agree - as the evolution of the rule logic slowly faded away from focusing on SIG requests, the name and description did not follow - will update 👍